dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -514,6 +541,15 @@ def _update_version_in_provider_yaml(
         v = v.bump_patch()
     elif type_of_change == TypeOfChange.MISC:
         v = v.bump_patch()
+
+    get_console().print(f"[special]Bumping version to {v}...\n")
+
+    if _version_exists_in_changelog(provider_id, str(v)):
+        get_console().print(
+            f"[error]Version {v} already exists in CHANGELOG. Fix manually before proceeding.[/]"
+        )
+        exit(1)
```

Comment:
This will exit the process?
If detected we should show the message and let release manager fix it, once fixed it should continue from the same place. Basically,  don't exit the process just redo the command for this provider after release manager fixed the issue.
Looking for something like:

```
Version {v} already exists in CHANGELOG. Fix manually before proceeding.
Click Y for redo this provider after fix, S to skip this provider, Q to exist the process.
```